test/fn/upload-custom-translations.spec.js

```diff
@@ -434,7 +434,17 @@ describe('upload-custom-translations', () => {
       });
 
     });
+    
 
+  });
 
+  describe('medic-x.x', () => {
```

Comment:
what is this medic-x.x grouping? is it required?

Reply:
not required .. will remove it